sample/wear/build.gradle.kts

```diff
@@ -32,6 +32,7 @@ android {
     namespace = "com.datadog.android.wear.sample"
 
     compileOptions {
+        isCoreLibraryDesugaringEnabled = true
```

Comment:
so the only channel now to check how this Otel module behaves without desugaring enabled on the pre-24 APIs is now oncall app?

Reply:
nope also in Shopist...we have the Shopist running on 3 different device models